test/embedding.jl

```diff
@@ -0,0 +1,12 @@
+# test that the embedding example runs without error
+let
+    if is_windows()
+        embedding_bin = joinpath(JULIA_HOME,"embedding.exe")
+    else
+        embedding_bin = joinpath(JULIA_HOME,"..","libexec","embedding")
```

Comment:
it may be preferable to do the same thing on windows too for the same reasons, just use withenv to temporarily append to path to include wherever libjulia is when running it in the test